index.js

```diff
@@ -6,4 +7,9 @@ app.get('/', (req, res) => {
     res.send('La Bestia does the best readmes ever');
 })
 
+app.get('/interviews', interviews.listInterviews)
+app.get('/interviews/create', interviews.createInterview)
+app.get('/interviews/update', interviews.updateInterview)
+app.get('/interviews/remove', interviews.removeInterview)
```

Comment:
The team decided to use delete instead of remove. 

https://rrcourses.slack.com/archives/C02JCT04R44/p1636301668012900